sources/web/datalab/polymer/components/file-browser/file-browser.ts

```diff
@@ -142,42 +157,26 @@ class FileBrowserElement extends Polymer.Element {
 
     super.ready();
 
+    if (!this.fileManagerType) {
+      this.fileManagerType = 'jupyter';
+    }
+    switch (this.fileManagerType) {
+      case 'bigquery':
+        this._fileManager =
+            FileManagerFactory.getInstanceForType(FileManagerType.BIG_QUERY);
+        break;
+      default:
```

Comment:
Let's throw an error here. If the attribute is not specified it gets populated up on line 160, so this catches bad values.